Makefile

```diff
@@ -1,11 +1,28 @@
-all: index.html
+SHELL=/bin/bash
 
-force:
-	bikeshed -f spec ./index.src.html
+.PHONY: ci clean local remote
 
-index.html: index.src.html
-	bikeshed -f spec ./index.src.html
+local: index.bs
+	bikeshed --die-on=warning spec index.bs index.html
 
-publish:
-	git push origin master master:gh-pages
+remote: index.html
 
+ci: index.bs index.html
+	mkdir -p out
+	cp index.html out/
+
+clean:
+	rm index.html
+
+index.html: index.bs
+	@ (HTTP_STATUS=$$(curl https://api.csswg.org/bikeshed/ \
+	                       --output index.html \
+	                       --write-out "%{http_code}" \
+	                       --header "Accept: text/plain, text/html" \
+												 -F die-on=warning \
```

Comment:
Indents look off here

Reply:
That's really weird. I copied this directly from https://github.com/jeremyroman/alternate-loading-modes. Fixed.